L/LLVMExtra/build_tarballs.jl

```diff
@@ -0,0 +1,73 @@
+using BinaryBuilder, Pkg
+using Base.BinaryPlatforms
+
+include("../../fancy_toys.jl")
+
+name = "LLVMExtra"
+repo = "https://github.com/vchuravy/LLVMExtra.git"
+version = v"0.0.1"
+
+# Collection of sources required to build attr
+sources = [GitSource(repo, "30882a8aadea1e15178e6fc8a70165a88f8377b0")]
+
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd LLVMExtra
+CMAKE_FLAGS=()
+# Release build for best performance
+CMAKE_FLAGS+=(-DCMAKE_BUILD_TYPE=RelWithDebInfo)
+# Install things into $prefix
+CMAKE_FLAGS+=(-DCMAKE_INSTALL_PREFIX=${prefix})
+# Explicitly use our cmake toolchain file and tell CMake we're cross-compiling
+CMAKE_FLAGS+=(-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN})
+CMAKE_FLAGS+=(-DCMAKE_CROSSCOMPILING:BOOL=ON)
+# Tell CMake where LLVM is
+CMAKE_FLAGS+=(-DLLVM_DIR="${prefix}/lib/cmake/llvm")
+# Force linking against shared lib
+CMAKE_FLAGS+=(-DLLVM_LINK_LLVM_DYLIB=ON)
+# Build the library
+CMAKE_FLAGS+=(-DBUILD_SHARED_LIBS=ON)
+cmake -B build -S . -GNinja ${CMAKE_FLAGS[@]}
+ninja -C build -j ${nproc} install
+"""
+
+function configure(julia_version, llvm_version)
+    # These are the platforms we will build for by default, unless further
+    # platforms are passed in on the command line
+    platforms = expand_cxxstring_abis(supported_platforms())
```

Comment:
Would this work?
```suggestion
    platforms = expand_cxxstring_abis(supported_platforms(; experimental=julia_version>=v"1.7"))
```

Reply:
1.6 should also work no?

Reply:
Uh, I thought we didn't have LLVM 11 for those platforms, but we do, yes.
```suggestion
    platforms = expand_cxxstring_abis(supported_platforms(; experimental=true))
```